zello-sdk-sample/src/main/java/com/zello/sdk/sample/Helper.java

```diff
@@ -0,0 +1,31 @@
+package com.zello.sdk.sample;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+public class Helper {
+
+	private static final String _loginPref = "login";
+
+	public static void saveValue(Context context, String key, String data) {
+		if (context != null && key != null && key.length() >= 0) {
```

Comment:
Could you modify these methods to early return?

Reply:
Nice